typescript/packages/common-identity/src/pass-key.ts

```diff
@@ -15,11 +15,12 @@ const ALGS: PublicKeyCredentialParameters[] = [
   { type: "public-key", alg: -8 }, // ed25519
   { type: "public-key", alg: -7 }, // es256
   { type: "public-key", alg: -257 }, // rs256
-]
+];
 
 export interface PassKeyGetOptions {
-  mediation?: "conditional",
-  userVerification?: "required" | "preferred" | "discouraged"
+  mediation?: "conditional";
+  userVerification?: "required" | "preferred" | "discouraged";
+  allowCredentials?: PublicKeyCredentialDescriptor[];
```

Comment:
Exposed so I can pass in the key descriptor, allowing the user to skip choosing their key from a list of many